rasa/core/tracker_store.py

```diff
@@ -173,9 +173,23 @@ def create_tracker(
         return tracker
 
     def save(self, tracker):
-        """Save method that will be overridden by specific tracker"""
+        """Save method that will be overridden by specific tracker."""
         raise NotImplementedError()
 
+    def exists(self, sender_id: Text) -> bool:
```

Comment:
we are currently trying to consolidate all the appearances of `sender_id` in the code. Let's make sure new code uses `conversation_id` right away.
```suggestion
    def exists(self, conversation_id: Text) -> bool:
```